howto/faq/README.md

```diff
@@ -45,7 +45,12 @@ We learn about upcoming API breakages in UIKit as early as everyone else through
 
 ### How large is the upload size of the library?
 
-If you import all the components, the size of your app will increase by about 5.3MB. However, you don't have to add the entire library! You can add as many or as few of the 25+ individual components as you like. And that averages to less than .2MB per component.
+The size of different components vary from a just a few kibibytes (KiB) to a
+few hundred kibibytes.  If you import all the components, the size of your app
+will increase by less than 3 mebibytes. However, you don't have to add the
```

Comment:
Is this a typo `mebibytes`?

Reply:
I mean, they are real things but why not do the more colloquial KB and MB?

Reply:
Switched to 10^6 terms

Reply:
+1 to 'standard' KB and MB.  Our users may get curious and go internetting to find out what a kibibyte is rather than playing with our righteous code.